isodatetime/tests.py

```diff
@@ -1404,8 +1401,7 @@ def test_timepoint_at_year(test_year):
         test_data += data.get_days_since_1_ad(year - 1)
         assert_equal(test_data, ctrl_data)
         for attribute, attr_max in test_duration_attributes:
-            delta_attr = random.randrange(0, attr_max)
-            kwargs = {attribute: delta_attr}
+            kwargs = {attribute: random.randrange(0, attr_max)}
```

Comment:
Thank you codacy-bot, luckily this pseudo-random generator is not being used for either security of cryptographic purposes.